audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditService.java

```diff
@@ -37,80 +39,71 @@
  */
```

Comment:
sort out javadoc formatting for the class javadoc, line 34 needs to be split to actually represent the html tags specified

Reply:
Split out JavaDoc HTML tags